app/uk/gov/hmrc/traderservices/wiring/AppConfig.scala

```diff
@@ -63,15 +63,20 @@ trait AppConfig {
   val exitSurveyUrl: String
   def requestUri(implicit request: RequestHeader): String =
     SafeRedirectUrl(baseExternalCallbackUrl + request.uri).encodedUrl
+
   def feedbackUrl(implicit request: RequestHeader): String =
     s"$contactHost/contact/beta-feedback?service=$contactFormServiceIdentifier&backUrl=$requestUri"
 
   val signOutUrl: String
-  val researchBannerUrl: String
 
+  val researchBannerUrl: String
   val fileFormats: AppConfig.FileFormats
 
   val traceFSM: Boolean = false
+
+  val timeout: Int
+  val countdown: Int
+  val timedoutUrl: String
```

Comment:
we don't need this to be configured, the `timedout` URL is stable and can be referenced using 
 `appConfig.baseExternalCallbackUrl + routes.SessionController.showTimeoutPage.url`